backend/users/views/user.py

```diff
@@ -259,7 +259,7 @@ class UserAwaitingApprovalViewSet(
     filterset_class = UserFilter
 
     def get_queryset(self):
-        return self.user_model.objects.filter(channel=None).exclude(username="deleted")
+        return self.user_model.objects.filter(channel=None).exclude(username="deleted").exclude(username="external")
```

Comment:
Perhaps we could use the [`in` field lookup](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#in).

This would allow us to write the exclusions as a list we can give a name to, which is useful for documenting its purpose and reusing it elsewhere.

```suggestion
        VIRTUAL_USERS=["deleted", "external"]
        return self.user_model.objects.filter(channel=None).exclude(username__in=VIRTUAL_USERS)
```

`VIRTUAL_USERS` is not a very good name and the list probably shouldn't be declared there but you get what I mean